truffle/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "Lightchain-truffle",
-  "version": "0.0.1",
+  "name": "Lightchain-Truffle",
+  "version": "1.0.0",
   "description": "Servers as truffle suite and support for JS based parts of Lightchain codebase.",
   "scripts": {
-    "test-mainnet": "truffle test --reset --network=mainnet",
-    "test-sirius": "truffle test --reset --network=sirius",
-    "test-standalone": "truffle test --reset --network=standalone"
+    "test-mainnet": "./node_modules/.bin/truffle test --reset --network=mainnet",
```

Comment:
By default whe you run `npm` scripts this includes `./node_modules/.bin` as part of PATHs. So please, remove `./node_modules/.bin`

Reply:
oh nice didn't know, i put it back.